### PR TITLE
Set Up project

### PR DESCRIPTION
**Pull request by alvelur**
The basic structure of the project has been added

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>ReiAutomationSchool</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>15</maven.compiler.source>
```
Comment (juanfer345) [inline pom.xml:12]: Didn't put versions as parameters 😠!!

File: ReiAutomationSchool.iml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (juanfer345) [inline ReiAutomationSchool.iml]: This should be ignored

**Review by juanfer345 — CHANGES_REQUESTED** [review]
Looks good, but some changes are necessary

File: pom.xml (hunk 1 above)
Comment (alvelur) [reply pom.xml:12]: Done!

File: ReiAutomationSchool.iml (hunk 2 above)
Comment (alvelur) [reply ReiAutomationSchool.iml]: Done!

File: pom.xml (hunk 1 above)
Comment (juanfer345) [reply pom.xml:12]: Noice!

**Review by juanfer345 — APPROVED** [review]

**Review by juanfer345 — APPROVED** [review]

**Pull request merged**